tests/fixtures/end_to_end_tests_measures/measures_cohort.py

```diff
@@ -1,12 +1,14 @@
-from cohortextractor import Measure, table
+from cohortextractor import Measure, codelist, table
 
 
 class Cohort:
     _clinical_events = table("clinical_events")
     _registrations = table("practice_registrations")
     population = _registrations.exists()
     practice = _registrations.first_by("patient_id").get("pseudo_id")
-    has_event = _clinical_events.filter(code="abc").first_by("patient_id")
+    has_event = _clinical_events.filter(code=codelist(["abc"], system="ctv3")).first_by(
```

Comment:
I wonder if we should provide `ctv3(*codes)` and `snomed(*codes)` functions as sugar here? Maybe it can wait until we iterate on the DSL.